configure

```diff
@@ -732,15 +737,39 @@ else
 	fi;
 fi
 
-if [ -z "$systemdsystemunitdir" ]; then
-	if $PKG_CONFIG --exists systemd; then
-		systemdsystemunitdir=$($PKG_CONFIG --variable=systemdsystemunitdir systemd)
-	fi
+if [ ! "$systemd" = 0 ]; then
```

Comment:
if systemd is not set, where does it get set to 1?

Reply:
In the top where all command line options are set.